<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="4">
      <widget class="QProgressBar" name="progress"/>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="startStop">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frequency minimum:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QuantityWidgetController" name="frequencyMinimum"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Generator power:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="PowerWidgetController" name="generatorPower"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frequency maximum:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QuantityWidgetController" name="frequencyMaximum"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Frequency steps:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Logaritmic sweep:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="logarithmic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="IntegerWidgetController" name="frequencySteps"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DropWidget" name="transmittedPower" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuantityWidgetController</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui.quantitywidget_controller.h</header>
  </customwidget>
  <customwidget>
   <class>DropWidget</class>
   <extends>QWidget</extends>
   <header>gui.dropwidget_controller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PowerWidgetController</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui.powerwidget_controller.h</header>
  </customwidget>
  <customwidget>
   <class>IntegerWidgetController</class>
   <extends>QSpinBox</extends>
   <header>gui.integerwidget_controller.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
